<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1118</width>
    <height>822</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>10</y>
     <width>258</width>
     <height>275</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="3">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>to</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>G</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLineEdit" name="BH">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>51</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="GL"/>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="GH">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>51</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="RL">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>B</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>to</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>to</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="thresholding">
        <property name="text">
         <string>阈值提取</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="BL"/>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="RH">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>51</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="thresholdinglabel">
      <property name="minimumSize">
       <size>
        <width>256</width>
        <height>180</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>256</width>
        <height>180</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>0</y>
     <width>788</width>
     <height>216</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QPushButton" name="choosepicture">
        <property name="text">
         <string>导入图片</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>256</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>256</width>
          <height>180</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QComboBox" name="show_color_comboBox">
          <item>
           <property name="text">
            <string>R</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>G</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>B</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="singleshow">
          <property name="text">
           <string>单波段显示</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="singleshowlabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>201</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>256</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>256</width>
          <height>180</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>R</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>G</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>B</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="show_histogram">
          <property name="text">
           <string>显示直方图</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="histogramlabel">
        <property name="minimumSize">
         <size>
          <width>256</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>256</width>
          <height>180</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>714</width>
     <height>560</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="bayes_back">
            <property name="text">
             <string>选取背景</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bayes_start">
            <property name="text">
             <string>选取物体</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="bayes_choose_label">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>240</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>240</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QComboBox" name="Bayes_comboBox">
            <item>
             <property name="text">
              <string>R</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>G</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ALL</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Bayes_button">
            <property name="text">
             <string>贝叶斯分类</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="bayes_show_label">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>240</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>240</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="fisher_back">
            <property name="text">
             <string>选取背景</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="fisher_start">
            <property name="text">
             <string>选取物体</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="fisher_choose_label">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>240</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>240</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QComboBox" name="Fisher_comboBox">
            <item>
             <property name="text">
              <string>R</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>G</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ALL</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="fisher_button">
            <property name="text">
             <string>Fisher线性判别</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="fisher_show_label">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>240</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>240</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
